server/src/test/java/org/elasticsearch/index/mapper/RangeFieldMapperTests.java

```diff
@@ -344,4 +346,29 @@ public void testIllegalFormatField() throws Exception {
         );
         assertThat(e.getMessage(), containsString("Invalid format: [[test_format]]: Unknown pattern letter: t"));
     }
+
+    @Override
+    protected Supplier<? extends Object> randomFetchTestValueVendor(MappedFieldType ft) {
+        RangeType rt = ((RangeFieldType) ft).rangeType();
+        // Doc value fetching crashes.
+        // https://github.com/elastic/elasticsearch/issues/70269
+        assumeFalse("DocValuesFetcher doesn't work", true);
+        if (rt.isNumeric()) {
+            Supplier<Number> numbers = randomFetchValueVendor(rt.numberType().numericType());
+            return () -> {
+                Number lhs = numbers.get();
+                Number rhs = numbers.get();
+                // All the number subclasses we get are comparable to each other
+                @SuppressWarnings("unchecked")
+                Comparable<Number> hack = (Comparable<Number>) lhs;
+                if (hack.compareTo(rhs) > 0) {
+                    Number swap = lhs;
+                    lhs = rhs;
+                    rhs = swap;
+                }
+                return Map.of(randomBoolean() ? "gt" : "gte", lhs, randomBoolean() ? "lt" : "lte", rhs);
+            };
+        }
+        throw new UnsupportedOperationException("unsupported field type: " + ft);
```

Comment:
Maybe leave a TODO comment for IP and Date ranges?

Reply:
I wanted to come up with some way to make sure we don't throw it away for when we're ready for it. I can revert it from the PR though because I can use the revert commit in the PR to revive it.